lms/djangoapps/commerce/tests/__init__.py

```diff
@@ -22,21 +23,25 @@
 }
 
 
+@override_settings(ECOMMERCE_API_SIGNING_KEY=TEST_API_SIGNING_KEY, ECOMMERCE_API_URL=TEST_API_URL)
 class EcommerceApiClientTest(TestCase):
     """ Tests to ensure the client is initialized properly. """
 
     TEST_USER_EMAIL = 'test@example.com'
     TEST_CLIENT_ID = 'test-client-id'
 
-    @override_settings(ECOMMERCE_API_SIGNING_KEY=TEST_API_SIGNING_KEY, ECOMMERCE_API_URL=TEST_API_URL)
+    def setUp(self):
+        self.user = UserFactory()
+        self.user.email = self.TEST_USER_EMAIL
+        self.user.save()  # pylint: disable=no-member
+
+
```

Comment:
thanks for calling that out, pushing fix.   tho, the build seems to have died for other reasons afaict - https://build.testeng.edx.org/job/edx-platform-all-tests-pr-flow/6202/